app/migrations/Version20160114000000.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @package     Mautic
+ * @copyright   2015 Mautic Contributors. All rights reserved.
+ * @author      Mautic
+ * @link        http://mautic.org
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Migrations\SkipMigrationException;
+use Doctrine\DBAL\Schema\Schema;
+use Mautic\CoreBundle\Doctrine\AbstractMauticMigration;
+use OpenCloud\Queues\Resource\Message;
+
+/**
+ * Class Version20160114000000
+ */
+
+class Version20160114000000 extends AbstractMauticMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function mysqlUp(Schema $schema)
+    {
+        $this->addSql('ALTER TABLE ' . $this->prefix.'form_fields ADD COLUMN is_auto_fill TINYINT(1) NOT NULL DEFAULT 0');
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function postgresqlUp(Schema $schema)
+    {
+        $this->addSql('ALTER TABLE ' . $this->prefix . 'form_fields ADD COLUMN is_auto_fill TINYINT(1) NOT NULL DEFAULT 0');
```

Comment:
Don't think this would work for PostgreSQL. Doctrine's boolean field translates to a boolean type for pg. See https://github.com/mautic/mautic/blob/c3b256a961fb03f3eea38df18a310d5dab623715/app/migrations/Version20150724000000.php#L137-L137 for an example.